network/hostnetwork/transport_test.go

```diff
@@ -122,6 +126,7 @@ func TestNewInternalTransport2(t *testing.T) {
 	tp, err := NewInternalTransport(mockConfiguration("127.0.0.1:0"), ID1+DOMAIN)
 	require.NoError(t, err)
 	go tp.Start(ctx)
+	time.Sleep(time.Millisecond)
```

Comment:
I'm not convinced this is the right way to do whatever the author wanted to do. Will not work on Travis.